### ISSUE TITLE
Tags can not be empty error with version 1.10.3

### ISSUE THREAD

**Issue by amilagm** [fabfuel/ecs-deploy#146]
Hi,

We use ecs-deploy to deploy a react app to fargate and this morning our pipeline started failing with the following error.

`botocore.errorfactory.ClientException: An error occurred (ClientException) when calling the RegisterTaskDefinition operation: Tags can not be empty.
`

I noticed that there was a new version published to pypi yesterday so I changed the pipeline to use the previous version 1.10.2 and it started working again. So I assume this to be an issue with the 1.10.3 release.

Here are the environment and exception details:

```
ci pipeline : gitlab
executor: kubernetes
image: docker:latest
aws-cli: 1.18.142
```

Exception from `ecs deploy`:

```
Creating new task definition revision
Traceback (most recent call last):
  File "/usr/bin/ecs", line 11, in <module>
    load_entry_point('ecs-deploy==1.10.3', 'console_scripts', 'ecs')()
  File "/usr/lib/python3.8/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.8/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.8/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/ecs_deploy/cli.py", line 94, in deploy
    new_td = create_task_definition(deployment, td)
  File "/usr/lib/python3.8/site-packages/ecs_deploy/cli.py", line 474, in create_task_definition
    new_td = action.update_task_definition(task_definition)
  File "/usr/lib/python3.8/site-packages/ecs_deploy/ecs.py", line 599, in update_task_definition
    response = self._client.register_task_definition(
  File "/usr/lib/python3.8/site-packages/ecs_deploy/ecs.py", line 64, in register_task_definition
    return self.boto.register_task_definition(
  File "/usr/lib/python3.8/site-packages/botocore/client.py", line 337, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/usr/lib/python3.8/site-packages/botocore/client.py", line 656, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.ClientException: An error occurred (ClientException) when calling the RegisterTaskDefinition operation: Tags can not be empty.
```

**Comment by fabfuel** [comment]
Hi @amilagm! 

Sorry for the issue and thanks for the detailed bug report. I'll revoke the version on PyPI and provide a bugfix asap!

Best
Fabian

**Comment by amilagm** [comment]
Hi @fabfuel,

No worries, thank you very much!

**Comment by amilagm** [comment]
I can confirm that this is now fixed on the version `1.10.5`.

@fabfuel  Thanks for fixing quickly.
